src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+abstract class Figure implements FigureDraw {
+    protected Color color;

```

Comment:
The 'color' variable should be private and have getters and setters to maintain encapsulation principle of OOP.